test_snakes_cafe.py

```diff
@@ -1,22 +1,28 @@
 import snakes_cafe as sc
-import pytest as pt
+# import pytest
 
 
 def test_add_item_returns_string():
-    # test add_item function is recieving expected input 
```

Comment:
This test naming and doc string do not line up with what is actually being asserted which right now is the default receipt.